ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/management-interfaces.cfg

```diff
@@ -7,6 +7,10 @@ hostname management-interfaces
 no enable password
 no aaa root
 !
+interface Management0
```

Comment:
Missing changes in documentation - cf https://github.com/aristanetworks/ansible-avd/actions/runs/3464392573/jobs/5785922148#step:5:1436

Did you re run the molecule scenario?